pyproject.toml

```diff
@@ -13,7 +13,8 @@ repo-manager = "repo_manager.main:main"
 
 [tool.poetry.dependencies]
 python = "^3.11"
-pydantic = "^1.10.9"
+pydantic = "^2.7.1"
```

Comment:
pydantic 2 is a major breaking change and all the schemas would have to be updated to support it.

If you want to do that, I support it, but it should be in its own PR and all schemas updated to use the new v2 syntax

Reply:
@andrewthetechie  see #225